### PR TITLE
Finish repo init

### PR DESCRIPTION
**Pull request by oriolgual**
Imported some files from the old repo and added GitHub templates

### CODE REVIEW

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -8,20 +8,20 @@
...
-#     endpoint:              'https://s3.example.com:8080'                      # optional, defaults to nil
-#   }
-#   config.fog_directory  = 'name_of_directory'                                 # required
-#   config.fog_public     = false                                               # optional, defaults to true
-#   config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" }    # optional, defaults to {}
-# end
+if Rails.env.production?
+  require 'carrierwave/storage/fog'
+
+  CarrierWave.configure do |config|
```
Comment (josepjaume) [inline config/initializers/carrierwave.rb]: I would copy this from https://github.com/HospitaletDeLlobregat/decidim-hospitalet/blob/master/config/initializers/carrierwave.rb as the newest carrierwave/fog/whatever versions gave me quite a hard time.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+For more informal communication, contact team members via twitter
+
+## Resolve an issue
+
+Admins tag issues using two label related with collaboration availability:
+
+* `PRs-welcome`: [issues labeled with PRs-welcome](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/labels/PRs-welcome) are well defined features ready to be implemented by whoever wants to do it.
+
+* `Not-ready`: with this label admins mark features or changes that are not well defined yet or subject to an internal decision. Is not a good idea to start implementation of these issues.
```
Comment (josepjaume) [inline CONTRIBUTING.md]: This is not exactly how we'll be organizing ourselves and it mainly inherits from consul. Should we just copy it for now and modify it later? (famous last words)

File: config/initializers/carrierwave.rb (hunk 1 above)
Comment (oriolgual) [inline config/initializers/carrierwave.rb]: This comes from hospitalet xD

File: CONTRIBUTING.md (hunk 2 above)
Comment (oriolgual) [inline CONTRIBUTING.md]: I'll remove it for now.

**Pull request merged**